packages/experimental-opentelemetry-web/src/user-interaction-instrumentation.ts

```diff
@@ -37,10 +37,53 @@ export class UserInteractionInstrumentation extends InstrumentationBase {
     // enable() gets called by our superclass constructor
     // @ts-expect-error this may get set in enable()
     this._listeners = this._listeners ?? [];
+    UserInteractionInstrumentation._eventMap = new WeakMap();
```

Comment:
Setting a static member from inside the constructor means that the map will be reset every time you instantiate a new `UserInteractionInstrumentation` instance, right? Is that a problem?

I assume you don't want to assume `UserInteractionInstrumentation` is a singleton, or else you wouldn't need to use a `static` in the first place.

Reply:
Pragmatically no since instrumentation should only get initialized on startup, BUT def not intended. Thanks for catching! I def forgot to clean this bit up from an earlier code-thought.

Reply:
See 1ced6b4